index.ts

```diff
@@ -164,13 +164,13 @@ function getVideos(doc: cheerio.Root) {
       const node = nodes[index];
       if (node.type === `tag`) video = node.attribs.content;
 
-      nodeType = nodeTypes![index];
-      if (nodeType.type === `tag`) {
+      nodeType = nodeTypes && nodeTypes[index];
```

Comment:
Doesn't the optional operator work?

```suggestion
      nodeType = nodeTypes?.[index];
```

Reply:
Yeah it should work!

Reply:
Thanks!